tests-web/assets/js/main.js

```diff
@@ -1,22 +1,39 @@
 import { makeDropHandler, makeLinkHandler, handleDragOver } from "./files.js";
+import { makeOpenTest } from "./tests.js";
 
-const cb = (map) => {
+const useMap = (map) => {
     const container = map.getContainer();
     container.ondrop = makeDropHandler(map);
     container.ondragover = handleDragOver;
-    console.info("Ready for drops on the map!", container);
 
     map.loadLink = makeLinkHandler(map);
+
+    console.info("New map created! File drops enabled.", container);
+
+    console.info("opening a test, just for fun...");
+    makeOpenTest(map)("aurora_sausage_link");
```

Comment:
![Screenshot from 2022-06-21 10-02-59](https://user-images.githubusercontent.com/1664407/174761450-69223b73-0406-4a08-baf5-d5fc80f07734.png)
Cool, it's working well! A few questions:

1) What file should be dropped into the browser window -- the geojson, dot, osm.xml?
2) Is the next idea to get a dropdown or some other way of browsing through test cases?
3) Why are there blue rectangles for the landuse areas -- is that the .osm.xml being parsed and displayed? I assume we'll also eventually have a way to toggle the 3 layers?

Reply:
1. Any file that is dropped will be added as a layer (based on it's file extension, if it's supported). This is a convenience for testing, and as a hassle free shortcut to viewing some file in the interface.
2. A list of tests is the intended interface, yeah. See #29.
3. The whole input.xml is being displayed!
	- The parsing of it is all done inline in the vendored `leaflet-osm.js`. We can customise that to our heart's content. (wishlist: style roads and road nodes distinctively, the rest as faint wireframe; crop features or mask layer to bounds specified in the file)
	- Layer controls will be needed to a certain extent, esp. for dropped files and the network graph. Hopefully we can do the right thing with styling to avoid most cases where fiddling with layers is required.